src/client/mixins/linked-in.ts

```diff
@@ -0,0 +1,28 @@
+import { Page } from 'puppeteer';
+
+export class LinkedInAwareMixin {
+  public client: Page;
+
+  async validateLinkedInInsightTag(pid: number) {
+    await (this as any).waitForNetworkIdle(10000, false);
+    const requests = await (this as any).getFinishedRequests();
+    const linkedInRequests = requests.filter(r => r.method == 'GET'
+                            && (r.url.startsWith('https://px.ads.linkedin.com/collect')
+                                || r.url.startsWith('https://dc.ads.linkedin.com/collect')));
+
+    const urls = linkedInRequests.map(l => l.url);
+    return Object.values(urls).join('').includes(`pid=${pid.toString()}`);
```

Comment:
- [ ] Recommend using node.js [URL](https://nodejs.org/api/url.html#url_url) and [SearchParams](https://nodejs.org/api/url.html#url_class_urlsearchparams) modules/classes to be more precise here.

Want to avoid situation where partner ID that fired was, e.g. `123987`, and someone was checking for partner ID `123`.  (False positive)

Reply:
nice